packages/studio/src/components/Highlighter.tsx

```diff
@@ -26,15 +29,21 @@ export default function Highlighter() {
     return {
       position: "absolute",
       zIndex: "10",
-      left: `${window.scrollX + rect.left}px`,
-      top: `${window.scrollY + rect.top}px`,
+      left: `${
+        (iframeEl?.contentWindow ? iframeEl.contentWindow.scrollX : 0) +
```

Comment:
nit but this (and below) can be `iframe?.contentWindow?.scrollX ?? 0`

Reply:
Oooh ur right! Thanks!